src/actions/transformMatch.js

```diff
@@ -155,9 +155,13 @@ function transformMatch(m) {
       dn_ten: (player.dn_t || [])[10],
       analysis: analyzeMatch(m, player),
     };
+
+    // compute a cs_t as a sum of lh_t & dn_t
+    player['cs_t'] = (player.lh_t || []).map((v, i) => v + (player.dn_t || [])[i]);
+
     // filter interval data to only be >= 0
     if (player.times) {
-      const intervals = ['lh_t', 'gold_t', 'xp_t', 'times'];
+      const intervals = ['lh_t', 'cs_t', 'dn_t', 'gold_t', 'xp_t', 'times'];
```

Comment:
I added the dn_t to the list of things to get filtered.

Reply:
And of course adding the `dn_t` threw an error because some matches don't have the dn_t data calculated.  Reverted.